sssd_test_framework/hosts/ad.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, *args, **kwargs) -> None:
         self.adminpw: str = self.config.get("adminpw", "Secret123")
         """Password of the Administrator user, defaults to ``Secret123``."""
 
+        self.adminuser: str = self.config.get("adminuser", "Administrator")
+        """Administrator user, defaults to ``Administrator``."""
```

Comment:
Ideally, this would go to a separate commit. Or in this case, even separate PR, as it is also present in the adcli patches. You will have to rebase one or the other when we merge.